site/en/docs/extensions/whatsnew/index.md

```diff
@@ -16,7 +16,13 @@ updated: 2022-01-20
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes.
 
-### 2022.01.05: `match_origin_as_fallback` in Canary {: #canary-match-origin-as-fallback }
+### Chrome 100: i18n.getMessage() support in extension service workers {: #m100-i18n-getmessage }
+
+The [`chrome.i18n.getMessage()`](/docs/extensions/reference/i18n/#method-getMessage) API is now
+supported in extension service worker contexts. This change landed in Canary on January 27, 2022
+and is expected to reach Stable in Chrome 100.
+
+### 2022.01.05: match_origin_as_fallback in Canary {: #canary-match-origin-as-fallback }
```

Comment:
Was removing the backticks in this title intentional?

Reply:
It was, but I don't think I made the right call. I noticed that I've been inconsistent about the use of &lt;code&gt; tags in headers, so I dropped them to be more consistent. For the moment, I'm going to reverse course on this and add it to a couple other headers that didn't use them in the past.